index.html

```diff
@@ -1956,39 +1956,9 @@ <h3>Securing Verifiable Credentials</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
```

Comment:
presentations have not ever required at least one proof historically, I believe this was accidentally changed in a previous PR.

Reply:
do they have any utility without a proof? are they even verifiable (no)?